docs/static/upgrading.asciidoc

```diff
@@ -70,26 +70,27 @@ If you are installing Logstash with other components in the Elastic Stack, also
-Logstash 2.x instance via the default HTTP communication layer. This provides some flexibility to decide when to upgrade
+Elasticsearch 6.0 does not require Logstash 6.0. An Elasticsearch 6.0 cluster will happily receive data from a
+Logstash 5.x instance via the default HTTP communication layer. This provides some flexibility to decide when to upgrade
 Logstash relative to an Elasticsearch upgrade. It may or may not be convenient for you to upgrade them together, and it
-is
-not required to be done at the same time as long as Elasticsearch is upgraded first.
+is not required to be done at the same time as long as Elasticsearch is upgraded first.
 
-You should upgrade in a timely manner to get the performance improvements that come with Logstash 5.0, but do so in
+You should upgrade in a timely manner to get the performance improvements that come with Logstash 6.0, but do so in
 the way that makes the most sense for your environment.
 
 ==== When Not to Upgrade
 
-If any Logstash plugin that you require is not compatible with Logstash 5.0, then you should wait until it is ready
+If any Logstash plugin that you require is not compatible with Logstash 6.0, then you should wait until it is ready
 before upgrading.
 
-Although we make great efforts to ensure compatibility, Logstash 5.0 is not completely backwards compatible. As noted
-in the Elastic Stack upgrade guide, Logstash 5.0 should not be upgraded before Elasticsearch 5.0. This is both
-practical and because some Logstash 5.0 plugins may attempt to use features of Elasticsearch 5.0 that did not exist
-in earlier versions. For example, if you attempt to send the 5.x template to a cluster before Elasticsearch 5.0, then it
-will not be able to use it and all indexing will fail likely fail. If you use your own, custom template with Logstash,
-then this issue can be ignored.
+Although we make great efforts to ensure compatibility, Logstash 6.0 is not completely backwards compatible. As noted
+in the Elastic Stack upgrade guide, Logstash 6.0 should not be upgraded before Elasticsearch 6.0. This is both
+practical and because some Logstash 6.0 plugins may attempt to use features of Elasticsearch 6.0 that did not exist
+in earlier versions.
 
-Note the Elasticsearch Output Index Template change in the <<breaking-changes>> documentation for further insight into
-this change and how it impacts operations.
+Note that multiple doctypes are no longer supported in Elasticsearch 6.0. Please refer to
+ {ref}/removal-of-types.html[Removal of mapping types] and {ref}/breaking-changes.html[Breaking changes] for more information.
+
+To continue using default templates after migrating Elasticsearch to 6.0, the existing template must be overwritten with the 6.x template.
+ This can be done by starting a pipeline with the overwrite_template => true option in the Elasticsearch output definition.
+  If you are using a custom mapping template, you may need to update it in a similar manner to reflect that.
```

Comment:
Language here is unclear. The phrase "you may need" suggests uncertainty. How can users know whether their custom mapping template needs to be upgraded? And what do you mean by "update it in a similar manner"? We need to add more guidance here, or users are unlikely to know what to do.